app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/liability_details.scala.html

```diff
@@ -47,11 +47,13 @@
 
             <h3 class="heading-medium mt-0">@messages("case.liability.about.title") </h3>
 
-            <div id="edit-liability-details" class="grid-row  mb-2">
-                <span class="column-one-third">
-                    <a href="@routes.LiabilityController.editLiabilityDetails(c.reference)"> Edit details </a>
-                </span>
-            </div>
+            @if(c.status == CaseStatus.OPEN || c.status == CaseStatus.NEW && request.hasPermission(Permission.EDIT_LIABILITY)) {
```

Comment:
Can these details not be edited when it's in referred status?

Reply:
I think referred usually mean "unactionable" so I would presume no

Reply:
exactly, when we refer a case then we wait for action ( unattended ) to be committed meanwhile the case is on hold as application perspective.  Once this action is finished the same operator will reopen the case.